File: templates/web/backend/python/flask/sources/health.py
from datetime import datetime, timezone

from flask import Blueprint, jsonify

bp = Blueprint("health", __name__)


@bp.route("/health")
def health():
    return jsonify({
        "status": "ok",
        "service": "{{project_name}}",
        "timestamp": datetime.now(timezone.utc).isoformat(),
    })

File: templates/web/backend/rust/axum/sources/health.rs
use std::sync::Arc;

use axum::{extract::State, routing::get, Json, Router};
use chrono::Utc;
use serde_json::{json, Value};

use crate::config::Config;

pub fn router() -> Router<Arc<Config>> {
    Router::new().route("/health", get(health_handler))
}

async fn health_handler(State(cfg): State<Arc<Config>>) -> Json<Value> {
    Json(json!({
        "status": "ok",
        "service": cfg.name,
        "timestamp": Utc::now().to_rfc3339()
    }))
}
